builders/deploy.yaml

```diff
@@ -20,6 +20,15 @@ substitutions:
   _REGION:
   _REPO: 'github.com/google/exposure-notifications-server'
   _TAG:
+  _SKIP_DEPLOY_CLEANUP_EXPOSURE: false
```

Comment:
This feels... like a lot of maintenance. Can we make this a comma-separated list instead and then do a substring match?

Reply:
That would be a good idea, working on it